### DISCUSSION TITLE
Manually add Danfoss Ally 014G2461 to HA

### DISCUSSION THREAD

**Discussion by jeppper** [Koenkk/zigbee2mqtt#11815] [Devices]
Can someone tell me how to add the ALlY thermostat to the Home Assistant config? This is no longer mentioned for the device in the supported device list.
 
Like this for a sensor.

  - platform: "mqtt"
    name: "temperature_sensor"
    state_topic: "zigbee2mqtt/hvac"
    availability_topic: "zigbee2mqtt/bridge/state"
    unit_of_measurement: "°C"
    value_template: "{{ value_json.temperature }}"
    device_class: "temperature"

**Comment by jeppper** [comment]
Anyone has an idea?

**Comment by jeppper** [comment]
No one has an idea to this?

**Comment by rsporsche** [comment]
Just curious, why do you want to add it manually? Mine show up automatically in HA so I haven't felt the need.

I can share the HA mqtt discovery messages from my POPP 701721 thermostats (same as Danfoss Ally) if that would help.

**Reply by jeppper** [reply]
I just want to some specific attributes added and not all of them.
It always worked fine adding components manually but Z2M no longer shows the specific lines for adding them to HA.
If possible please show the discovery messages. Maybe it could help.

**Reply by rsporsche** [reply]
Actually it just occurred to me I just used an mqtt client (mqtt explorer for windows) to view the messages being sent my zigbee2mqtt, I assume that will be what you need, so might be easer to do that because the HA configs are split across many different domains (climate, number, sensor...)

Happy to help if I can though.

**Reply by jeppper** [reply]
Did you then manually add the different domains into HA anyway after observing the mqtt traffic?
